tapir/translations/locale/de/LC_MESSAGES/django.po

```diff
@@ -45,7 +45,7 @@ msgstr "Es gibt bereits eine Nutzer*in mit diesem Namen."
 
 #: accounts/models.py:155
 msgid "Displayed name"
-msgstr "Angezeigte Name"
+msgstr "Angezeigter Name"
```

Comment:
Does the change in this file relate to the topic of this PR? Seems not. Maybe they could be merged as a separate PR?

Reply:
No this is an old one

Reply:
Actually not. But it would be good to release it, 'cause I fixed some little stupid language bugs in german translation.

Reply:
To this: we would have also to discuss in the future who takes care about the django.po file and makemessages command, so that the translations are up to date.

Reply:
Yes, good point, we should keep the file up to date. And these are small changes, so it's okay that they will be included like you did.